README.md

```diff
@@ -19,63 +19,59 @@ Functions offered:
+  Assigns ownership of attribute given as argument "from_own" to alias given as argument "to_own" to type given as argument "type". Returns qid attached to the constraint.
 
 * relates(type: str, role: str):
 
-  Adds a role given as argument "role" to a relationship type given as argument "type". Returns qid attached to the query.
+  Adds a role given as argument "role" to a relationship type given as argument "type". Returns qid attached to the constraint.
 
 * relates_as(type: str, to_role: str, from_role: str):
 
-  Adds a role given as argument "from_role" to an alias given as argument "to_role" to type given as argument "type". Returns qid attached to the query.
+  Adds a role given as argument "from_role" to an alias given as argument "to_role" to type given as argument "type". Returns qid attached to the constraint.
 
 * plays(type: str, relation: str, role: str):
 
-  Assigns the relation:role, where relation is given as argument "relation" and role is given as argument "role" to the type given as argument "type". Returns qid attached to the query.
+  Assigns the relation:role, where relation is given as argument "relation" and role is given as argument "role" to the type given as argument "type". Returns qid attached to the constraint.
 
 * plays_as(type: str, relation: str, to_role: str, from_role: str):
 
-  Assigns the relation:role, where relation is given as argument "relation" and role is given as argument "from_role" to alias given as "to_role" to the type given as argument "type". Returns qid attached to the query.
+  Assigns the relation:role, where relation is given as argument "relation" and role is given as argument "from_role" to alias given as "to_role" to the type given as argument "type". Returns qid attached to the constraint.
 
 * regex(type: str, regex: str):
 
-  Adds a regex pattern given as argument "regex" to attribute type given as argument "type". Returns qid attached to the query.
+  Adds a regex pattern given as argument "regex" to attribute type given as argument "type". Returns qid attached to the constraint.
 
 * key(type: str, attribute: str):
 
   Makes the attribute given as argument "attribute" that is owned by type given as argument "type" a @key attribute.
 
 * unique(self, type: str, attribute: str):
 
-  Makes the attribute given as an argument "attribute" that is owned by type given as argument "type" a @unique attribute. Returns qid attached to the query.
+  Makes the attribute given as an argument "attribute" that is owned by type given as argument "type" a @unique attribute. Returns qid attached to the constraint.
 
-* print_query_log():
+* print_constraint_log():
 
-  Prints all the query IDs attached to every query.
-
-* remove(q_ids: list):
-
-  Removes all the queries given in the list argument "q_ids". And re-renders the remaining queries.
```

Comment:
Why are these removed by this PR? This work is disconnected from #5, it should be based off the current `master` branch, not off your feature branch - this is a great example of why using feature branches (which I notice you have now done) when working on multiple features is a good idea, it prevents you from inadvertently crossing the streams of your work.

This also applies to the changes in `test_builder.py`.